bundler/src/main/java/com/f2prateek/bundler/FragmentBundler.java

```diff
@@ -0,0 +1,37 @@
+package com.f2prateek.bundler;
+
+
+import android.app.Fragment;
+
+/**
+ * Set fragment arguments like a Pro.
+ * @param <F> The fragment to setup.
+ */
+public class FragmentBundler<F extends Fragment> {
```

Comment:
I agree. Initially for some reason I had the idea that these needed to be 2 separate classes but looking again right now it doens't make sense and in fact it is confusing. I'll merge them together.